docs/input4MIPs_datasets_CMIP6Plus.html

```diff
@@ -21,7 +21,7 @@
 <title>Input4MIPs CMIP6Plus datasets</title>
 </head>
 <body>
-<p><h1>Input4MIPs CMIP6Plus datasets: v6.5.3</h1><p>
+<p><h1>Input4MIPs CMIP6Plus datasets: v0.1.0</h1><p>
```

Comment:
This is an evolution from the CMIP6 era, so I suggest we start at 6.5 (CMIP6Plus) and continue to iterate until we get our "final" versions, and then toggle to 7.0.0, after which I am certain we will continue to tweak, 7.0.1, 7.0.2, 7.1.0, etc - following a similar path that was in [CMIP6 Forcing Dataset Summary](https://docs.google.com/document/d/1pU9IiJvPJwRvIgVaSDdJ4O0Jeorv_2ekEtted34K9cA/edit#heading=h.w4kchhc266o3)

Reply:
Ok cool. I'll fix this by hand here, then sort out doing it properly as part of #59

Reply:
```suggestion
<p><h1>Input4MIPs CMIP6Plus datasets: v6.5.4</h1><p>
```